mmv1/third_party/terraform/utils/self_link_helpers.go

```diff
@@ -181,3 +181,14 @@ func GetRegionFromRegionalSelfLink(selfLink string) string {
 	}
 	return selfLink
 }
+
+func GetLocationFromOpName(opName string) string {
```

Comment:
Can you please move this function to the file tagslocation_operation.go, as it is only used there for now?